source/guides/actions/control-flow.md

```diff
@@ -153,7 +153,7 @@ module Web::Controllers::Dashboard
 end
 ```
 
-`#halt` accepts a HTTP status code as the first argument.
+`#halt` accepts an HTTP status code as the first argument.
```

Comment:
I think that could go either way - I (in American English) pronounce HTML as "aitch-tee-em-el" which indicates that "an" is proper.  See http://english.stackexchange.com/questions/25742/is-it-supposed-to-be-a-html-or-an-html for discussion.